### ISSUE TITLE
Select dropdown does not trigger with LG TV Controller in arrow navigation mode.

### ISSUE THREAD

**Issue by OnzeJermain** [mateussouzaweb/crunchyroll-webos#14]
Select dropdown does not trigger with LG TV Controller in arrow navigation mode.

Will this bug be fixed? Thanks for the app.

**Comment by mateussouzaweb** [comment]
The only possible fix on this issue is by replacing the select element with a custom like interface. Will take some time, but eventually yes, it will be fixed.

**Comment by mateussouzaweb** [comment]
@OnzeJermain I just published a new experimental dropdown to substitute select element. Could you please check if that works for you?

**Comment by OnzeJermain** [comment]
> @OnzeJermain I just published a new experimental dropdown to substitute select element. Could you please check if that works for you?

I tried it with the latest version on homebrew channel. It's not working for me. I tried all buttons on the remote.

**Comment by joaopedrosgs** [comment]
Same problem here

**Comment by mateussouzaweb** [comment]
I just published a new version (1.6.0), with compilation from typescript... see if that works now.

**Comment by arbarbosa1979** [comment]
I tried to navigate through the episodes using the remote control of the Smart TV LG 32lk615bpsb, it's not Smart Magic, but in my case only the OK button works, the arrows don't go anywhere, so it only works on remote magic?

**Comment by arbarbosa1979** [comment]
I was able to navigate normally using the remote control, but I could not select the video resolution, even clicking ok, the resolution was not changed, it remained in auto, no problems switching from one series to another or from one episode to another.

**Comment by Clessasaur** [comment]
Yep with latest update I could finally use arrows to navigate around, but same on not being able to change resolution. At least the Auto seems to default to 1080p vs the official VRV app that liked to set it to 240p for absolutely no reason.
